### PR TITLE
remove f.PrintObjectSpecificMessage

### PR DESCRIPTION
**Pull request by juanvallejo**
**Release note**:
```release-note
NONE
```

Mailing list discussion: https://groups.google.com/forum/#!topic/kubernetes-sig-cli/gZWg8Ulo3Ao

Removes the `PrintObjectSpecificMessage` helper in an effort to reduce printing stack complexity, and avoid special-casing resources.

### CODE REVIEW

**Review by soltysh — APPROVED** [review]
lgtm

**Comment by soltysh** [comment]
/ok-to-test

@kubernetes/sig-cli-pr-reviews fyi

**Review by droot — COMMENTED** [review]
/lgtm

**Comment by deads2k** [comment]
mailing list look positive. approving.

/approve
/hold

@juanvallejo please wait for https://github.com/kubernetes/kubernetes/pull/60117 to merge since it is larger and we're near the end.  After it merges feel free to remove this hold.

**Pull request merged**
